### PR TITLE
Updated readme and requirements

### PR DESCRIPTION
**Pull request by skylerwharton**
- Updated readme to include `add_seattle_data` manager command.
- Updated requirements.txt.

r @yoninachmany @HanaPearlman

### CODE REVIEW

**Comment by HanaPearlman** [comment]
LGTM!

File: requirements/common.txt [hunk 1]
```diff
@@ -26,11 +27,15 @@ MarkupSafe==0.23
 mccabe==0.4.0
 pep8==1.7.0
 psycopg2==2.6.1
+pyaml==16.11.4
```
Comment (yoninachmany) [inline requirements/common.txt]: where in the project is pyaml?

File: requirements/common.txt [hunk 2]
```diff
@@ -26,11 +27,15 @@ MarkupSafe==0.23
...
 pyflakes==1.0.0
 pytz==2016.3
+PyYAML==3.12
 ratelim==0.1.6
+redis==2.10.5
```
Comment (yoninachmany) [inline requirements/common.txt]: is this currently used in project?

File: requirements/common.txt [hunk 3]
```diff
@@ -26,11 +27,15 @@ MarkupSafe==0.23
...
 requests==2.9.1
 requirements==0.1
+rq==0.5.6
```
Comment (yoninachmany) [inline requirements/common.txt]: is this currently used in project?

File: requirements/common.txt (hunk 1 above)
Comment (skylerwharton) [reply requirements/common.txt]: Pyaml is used for parsing the Seattle data.

File: requirements/common.txt (hunk 2 above)
Comment (skylerwharton) [reply requirements/common.txt]: It's used in flask-base, so we'll need it eventually. I can remove it for now though if you want.

File: requirements/common.txt (hunk 3 above)
Comment (skylerwharton) [reply requirements/common.txt]: It's used in flask-base, so we'll need it eventually. I can remove it for now though if you want.

File: requirements/common.txt (hunk 1 above)
Comment (yoninachmany) [reply requirements/common.txt]: PyYaml (http://pyyaml.org/wiki/PyYAMLDocumentation) is used (`import yaml`), but pyaml is a "PyYAML-based python module to produce pretty and readable YAML-serialized data" that is not used here (https://pypi.python.org/pypi/pyaml).

Comment (skylerwharton) [reply requirements/common.txt]: Ok, sounds good. Removed.

File: requirements/common.txt (hunk 3 above)
Comment (skylerwharton) [reply requirements/common.txt]: Removed.

File: requirements/common.txt (hunk 2 above)
Comment (skylerwharton) [reply requirements/common.txt]: Removed.

**Review by skylerwharton — COMMENTED** [review]
@yoninachmany thoughts now?

**Pull request merged**